examples/plot_pvlaues.py

```diff
@@ -142,7 +142,8 @@
     stderrs[predictor] = EvokedArray(stderr, epochs_info, tmin)
     t_vals[predictor] = EvokedArray(t_val, epochs_info, tmin)
     p_vals[predictor] = p_val
-    p_vals_evoked[predictor] = EvokedArray(p_val*1e-6, epochs_info, tmin)
+    # transform p-values to Shannon information values (i.e., surprise values)
+    s_vals[predictor] = EvokedArray(-np.log2(p_val)*1e-6, epochs_info, tmin)
```

Comment:
one thing to keep in mind for the API: We're using evoked objects now, and if we whant to plot stuff later (e.g., using `plot_compare_evokeds`) values get automatically tranfomed to microvolt. There is probably a better way of doing this.

Reply:
@JoseAlanis you don't need to do this. You can control scaling behavior via parameters in plotting functions.

Reply:
can we change scalings parameters in `plot_compare_evokeds` too? The docstring says:
> ylim : dict | None
              ylim for plots (after scaling has been applied)

but I dont see where to chage the scaling

Reply:
Yes you are right, this function does not have scalings -- which is a problem of the function. Want to do a PR in MNE to add scalings?

Reply:
Aye